include/PianoRoll.h

```diff
@@ -160,6 +160,8 @@ protected slots:
 
 	void hidePattern( Pattern* pattern );
 
+	void selectRegionFromPixels( int x, int y );
```

Comment:
The `(int x, int y)` is confusing me a bit. Shouldn't it be more something like `(int x_start, x_end)`?